src/pages/files.js

```diff
@@ -29,9 +24,10 @@ export default class Files extends React.Component {
 
     // sessionFiles is optional and can be null (always on the client)
     const sessionFiles = get(req, 'session.files');
-    const { files, dependencies, recommendations } = await getFilesData(
+    const { files, dependencies, recommendations } = await getData({
```

Comment:
I'm confused with this. What's the rationale behind this change?

Reply:
It was just a small abstraction, I had `getFilesData` and `getProfileData` on several pages, I needed to modify their arguments, I find myself updating them in multiple files so I decided bring it all together as a single method in a single file.